\documentclass[11pt,oneside,letterpaper]{article}

% graphicx package, useful for including eps and pdf graphics
\usepackage{graphicx}
\DeclareGraphicsExtensions{.pdf,.png,.jpg}

% basic packages
\usepackage{color} 
\usepackage{parskip}
\usepackage{float}
\usepackage{hyperref}

% text layout
\usepackage{geometry}
\geometry{textwidth=15.25cm} % 15.25cm for single-space, 16.25cm for double-space
\geometry{textheight=22cm} % 22cm for single-space, 22.5cm for double-space

% helps to keep figures from being orphaned on a page by themselves
\renewcommand{\topfraction}{0.85}
\renewcommand{\textfraction}{0.1}

% bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,font=small]{caption}

% review layout with double-spacing
%\usepackage{setspace} 
%\doublespacing
%\captionsetup{labelfont=bf,labelsep=period,font=doublespacing}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}
%\renewcommand\citeleft{(}
%\renewcommand\citeright{)}
%\renewcommand\citeform[1]{\textsl{#1}}

% Remove brackets from numbering in list of References
\renewcommand\refname{\large References}
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

\usepackage{authblk}
\renewcommand\Authands{ \& }
\renewcommand\Authfont{\normalsize \bf}
\renewcommand\Affilfont{\small \normalfont}
\makeatletter
\renewcommand\AB@affilsepx{, \protect\Affilfont}
\makeatother

% notation
\usepackage{amsmath}
\usepackage{amssymb}


\begin{document}

\newgeometry{top=4cm}

Dear MBE editorial board,

Please find attached our extensively revised manuscript entitled ``Reassortment between influenza B lineages and the emergence of a co-adapted PB1-PB2-HA gene complex''.  This is a resubmission of manuscript number MBE-14-0575. The editorial assessment stated that:

\begin{quote}
The reviewers made many suggestions for the improvement of this paper, and I agree that these improvements are needed. 
The value of this manuscript lies primarily in the analytical techniques presented. 
The result is simple but of interest.

As both reviewers noted, the discussion consists mostly of rambling speculation. 
I found it hard to follow (particularly the part about balancing selection) and neither convincing nor very interesting. 
There is nothing wrong with just admitting that you don't understand the process behind this pattern and suggesting some experiments. 
The writing needs to be improved throughout.

The word fixation is often used in a confusing manner. 
For example, see line 28 page 5: ``We focus only on PB1, PB2 and HA lineage labels, since all other segments have fixed either the Vic or the Yam lineage.'' 
See also the sentence on lines 48-49 of that page.

The first line of the abstract and the first paragraph of the introduction make it sound like influenza B is an increasingly worrisome public health problem. 
Influenza B does not need to be hyped to be interesting, and this type of ``journalism'' can makes the reader wonder about the validity of everything that follows.
\end{quote}

We would like to thank the reviewers' and the editor's thorough and insightful comments.
We appreciate the time they have taken to review the manuscript and the acknowledgment of its value.
We agree that the discussion, as it was, contained mostly irrelevant information and speculation.
The discussion has been considerably slimmed down to the most essential information and reworded in places.
We have additionally altered the terminology that caused confusion, e.g. the word ``fixation'' is now explicitly refered to as a population genetics term and used only where applicable.
We have also toned down the wording with respect to the importance of influenza B viruses.


Point-by-point review responses are attached, as well as a diffed PDF detailing all the changes that have been made since the initial submission.

Sincerely,\\
Gytis Dudas

\restoregeometry

\newpage

\section*{Reviewer responses}

Original reviewer criticisms are in plain text.  Our responses follow in \textbf{bold}.  

%%% REVIEWER 1 %%%
\section*{Reviewer 1}
1. ``Could the authors provide an estimate of how many reassortment events have occurred in these trees in total, based on the no. of transitions?''

\textbf{We have added a new supplementary figure (Figure S10), which shows the mean number of transitions for each trait/label as well as their 95\% HPDs in each tree.
We advise to interpret the new figure with caution - many successful reassortment events we have observed involve more than one segment (see Figure 6) and thus these counts are not independent.}

2. ``It would be useful, as comparison, to include a Supplementary figure similar to Fig 2, but with branches colored to show reassortment events involving the other segments (PA, NP, NA, MP, NS -- obviously this is a lot of combinations and a lot of trees, so maybe just show a representative subset).''

\textbf{We believe that although this is an interesting point, it is outside the scope of the current manuscript.
Additionally, the complete reassortment history of all segments is available in Figure 6, which can be combined with Figure 2 to reveal the phylogenetic position of all reassorting lineages.
To avoid making excessive amounts of figures with all possible combinations of trees and traits we have made the MCC trees (including trait mapping) publicly available at: \url{https://github.com/evogytis/fluB/tree/master/data/mcc\%20trees}.}

3. ``I didn't find any accession numbers or description of the data set in the Supplement.  
It would be useful to know how geographically representative the data set is.  
It certainly is possible that in a spatially biased data set certain variants (as in those with mixed PB1-PB2-HA complexes) could be under-represented.''

\textbf{Apologies for this. We have a separate file listing all sequence accession numbers from primary (GISAID) and secondary (GenBank) datasets but forgot to include it in the initial submission.
These files are available at \url{https://github.com/evogytis/fluB/tree/master/data/acknowledgement\%20tables}.
To make it easier to judge how representative the sequence sampling was we have included two new supplementary figures (Figures S8 and S9) which show the geographic distribution of sequences in the primary and secondary datasets.}

4. ``It seems that most of the mixed PB1-PB2-HA complexes are found earlier in the tree, closer to the root, and become less frequent as time goes on.  Is this worth expanding upon?''

\textbf{The reviewer raises a very interesting point. We have addressed this on page 12: 
``Indeed, we observe that PB1--PB2 reassortants are the rarest and least persistent among mixed-lineage PB1-PB2-HA strains and have not been isolated in great numbers.
In fact most reassortants breaking the PB1-PB2-HA complex apart have occured in the past, close to the split of Vic and Yam lineages and have become very rare since.''
Unfortunately due to the low numbers of reassortments this is no more than speculation that we would rather not dwell on too much.}

5. ``The authors have focused on major reassortment events between Yam and Vic lineages.  
If the definition of reassortment is expanded to include sub-clades within the Yam and Vic lineages (as appear quite clearly on the PB1 and PB2 trees), does the pattern still hold?  
Or is this only occurring at the larger scale?''

\textbf{This is a very insightful observation. The unusual pattern of PB1-PB2-HA co-reassortment could potentially be caused via physical means (e.g. packaging signals) and thus should be observable at the sub-clade level.
The alternative is linkage caused by selection, where sub-clade reassortants occur, but are not selected against as long as ``pure'' PB1-PB2-HA complexes are maintained.
We attempted to address this in supplementary information by using SPR distances (Figure S4), but because sub-clades are necessarily more closely related to each other we find that there is too much phylogenetic noise to investigate this in a satisfactory way using the current data.
It is our opinion that addressing sub-clade reassortment would require the development of new methods and a more suitable dataset.}

6. ``p3, line 3: one of these should be PB2''

\textbf{This has been fixed. Thank you for spotting the mistake.}

7. ``p12, line 20-21: `do not appear to function well' sounds too virological; as we really have no idea about functional interactions, it's probably safer to stick with population dynamics and say 'do not appear to be fit'''

\textbf{We agree. It has been changed per reviewer's suggestion.}

%%% REVIEWER 2 %%%
\section*{Reviewer 2}

1.1. ``Importance of influenza B: There is no need to underline this in the first two paragraphs.
Influenza B is important, just not as clinically relevant as influenza A.
The authors state in the introduction that influenza B has increased in prevalence recently, but this is just the last influenza season, and the majority of influenza cases and influenza seasons are still dominated by influenza A.''

\textbf{This was pointed out by the editor too.
It has been addressed by toning down the language in the abstract and the introduction.}

2.1. ``Figure 5: If I understand this figure correctly, the authors are using the BEAST MCC tree of each segment, looking at a particular point in the past, and counting the number of extant lineages that would be associated either with the original 1983 Yamagata virus or the original 1983 Victoria virus.''

\textbf{Figure 5 shows the ratio at which Victoria and Yamagata sequences were isolated over time and is based on simply counting how many sequences within any given temporal bin and segment are derived from Victoria and Yamagata lineages.
It has caused confusion with reviewer 2 before, which we believe is due to our inter-changeable use of the word ``lineage'' when we actually mean ``sequence''.
This has been corrected in the text.}

2.2 ``I understand what the `recent past' part of this chart means.
It means, for example, that the PA, NP, NA, and MP segments circulating today are all derived from the Yamagata lineage.
But what does the 1984 to 1987 part of this chart mean?''

\textbf{The 1984-1987 part of the chart does not exist, because we have omitted influenza seasons 1983/1984 to 1986/1987 from the figure after reviewer 2 made this comment during the previous review.
This was done because we only had single complete genomes for seasons 1983/1984 to 1986/1987.
However, we are certain that both Victoria and Yamagata lineage segments were still co-circulating in the 1994/1995 flu season.}

``Is it really true that at this time the majority or all of these segments were Victoria-type?
If yes, does this mean that when Vic and Yam split the initial diversification was in PB1-PB2-HA and the PA, NP, NA, and MP segments were largely conserved during this time?
If yes, should we really call these Victoria-type in 1984, or are they simply the ancestral type before the split occurred?''

\textbf{We believe these questions were made in ensuing misinterpretation of Figure 5.}

2.4. ``Does sampling bias affect the results of Figure 5?''

\textbf{The apparent fixation of one lineage or the other will be completely robust to sampling.
In order to alleviate concerns about geographic sampling bias we have now included two additional supplementary Figures (Figures S8 and S9) which show the geographic distribution of the sequences in both primary and secondary datasets.}

2.5. ``I believe we have enough influenza B sequences to reconstruct the post-2005 part of this chart.'' 

\textbf{We are not sure what the reviewer meant with this question.
Figure 5 tracks the ratio of Victoria to Yamagata lineage sequences up to the 2011/2012 influenza season.}

2.6. ``How much confidence do we have in 1995? 1985?''

\textbf{The confidence in the ratios displayed in Figure 5 is completely dependent on the numbers of genomes sequenced every year.
We provide the genome counts for each year in each segment at the top of Figure 5.
Before 1997, there are fewer than 10 genomes per season, suggesting caution in interpreting these frequencies.
Post 1997, we have more confidence.}

3.1. ``Figure 8:  I could not understand this (granted, I am not a phylogeneticist).
But, something seems wrong here.
How can we compare PB1-PB1 TMRCAs to PB1-PB2 TMRCAs when PB1 and PB2 are not homologous?
I have probably misunderstood something here.''

\textbf{Figure 8 has caused confusion during previous review, which we could not address directly earlier, but have included an additional figure (supplementary Figure S17) and elaborated on the $\Delta$TMRCA method in response.
Briefly, in the absence of reassortment we expect every segment in the genome to have the same tree, i.e. the tree of every segment should recapitulate the ``virus'' tree (analogous to ``species'' trees in diploid population genetics), including the dates of nodes.
Due to population bottlenecks influenza viruses go through each year we expect strains isolated at any given time to have descended from a single recent virus genome.
This descent from a single genome should therefore be reflected in the TMRCAs of all segments, the only exception being reassortment, which will dramatically alter the TMRCAs of the reassorted segment tree with respect to the background onto which it is reassorting.
We understand that this is a rather complicated and non-intuitive concept to grasp and we are willing to elaborate further on the underlying principle of the method in the text if necessary.}

3.2. ``Are these TMRCAs of partial virus genomes?''

\textbf{We are not sure what the reviewer meant with this question.
We hope, however, that our description above and Figure S17 help to clarify this issue.}

3.3. ``Should the caption read `Normalized mean TMRCA statistics for different segment pairs'?''

\textbf{We agree, the wording in the caption for Figure 8 is slightly misleading. It has been changed accordingly.}

4.1. ``The discussion is long and somewhat disorganized.
I think the authors should spend some time improving the writing here, and shortening it.
The central theme of the paper is this PB1-PB2-HA complex, and I would structure the discussion to address the main questions surrounding this complex.  
Something like:

1.  Why are these segments so tightly linked at the genetic level?

1a.  Co-adaptation?
Possibly.
There is good functional evidence that PB1 and PB2 should be co-adapted.
There is some evidence, but much less, that HA should be co-adapted to PB1-PB2.

1b.  Simple divergence?
Possibly, especially if epistatic interactions cause an adapted PB1-PB2-HA complex to synergistically interact with the genotype defined by the other five segments.
But, we do see reassortment among the other five segments, so the evidence here is mixed.''

\textbf{It is difficult to determine whether divergence or co-adaptation have caused linkage between PB1, PB2 and HA segments, and thus we have decided to combine these two scenarios into a single paragraph under the ``Linkage between PB1, PB2 and HA gene segments'' section in the discussion.
Our speculation on what might be the cause of the linkage is contained in the ``Potential mechanisms for reassortment incompatibility'' section that follows.}

4.2. ``2. Is frequency-dependent selection allowing Flu B to maintain two HA types (in the same way that Flu A maintains two circulating subtypes in the population)?''

\textbf{We have decided to exclude superfluous speculation about hitchhiking and frequency-dependent selection on the HA segment in the discussion.}

4.3. ``3. Are Flu B population sizes big or small?''

\textbf{We are not sure whether the reviewer refers to 'census' or effective population sizes.
It is widely accepted that influenza B viruses have a larger effective population size than influenza A viruses, but it would difficult to find experts who would hazard a guess as to the 'census' population size for influenza B viruses.}

4.4. ``Could stochastic events have played a role in Flu B's population structure?
Does the lack of sequence data in the 1980s and early 1990s affect our conclusions?''

\textbf{The reviewer makes a very good point - population structure could indeed result in stochastic linkage between segments.
We have made note of this on page 12:
``Although the linkage between the two polymerase segments and HA is more difficult to explain, it is widely accepted that Victoria lineage HA had been restricted to eastern Asia between 1992 and 2000, offering ample time for the budding Victoria lineage to accumulate alleles causing reassortment incompatibility.
Without more genomic data from the past, however, it is difficult to estimate to what extent influenza B virus population structure contributed to the development of the current segment linkage.''}

4.5. ``4. Will influenza B speciate?''

\textbf{This paragraph was moved to its own section titled ``Will influenza B viruses speciate?'' and has been made more concise and focused.}

4.6. ``( Throughout the discussion `balancing selection' is used in a way I don't understand.
On page 15, if Vic and Yam are antigenically dissimilar, then it is frequency-dependent selection -- not balancing selection -- that prevents one genotype from driving the other to extinction. )''

\textbf{Apologies for the confusion, all references to balancing selection have been removed as irrelevant to the current manuscript.
We think that the maintenance of two subtypes in the influenza B population is an interesting question deserving of a separate discussion.}

\end{document}