\documentclass[11pt,oneside,letterpaper]{article}

% graphicx package, useful for including eps and pdf graphics
\usepackage{graphicx}
\usepackage{grffile}
%\DeclareGraphicsExtensions{.pdf,.png,.jpg}

% basic packages
\usepackage{color} 
\usepackage{parskip}
\usepackage{float}

% text layout
\usepackage{geometry}
\geometry{textwidth=15cm} % 15.25cm for single-space, 16.25cm for double-space
\geometry{textheight=22cm} % 22cm for single-space, 22.5cm for double-space

% helps to keep figures from being orphaned on a page by themselves
\renewcommand{\topfraction}{0.85}
\renewcommand{\textfraction}{0.1}

% bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,font=small]{caption}

% review layout with double-spacing
%\usepackage{setspace} 
%\doublespacing
%\captionsetup{labelfont=bf,labelsep=period,font=doublespacing}

% cite package, to clean up citations in the main text. Do not remove.
%\usepackage{cite}
\usepackage{natbib}
%\renewcommand\citepleft{(}
%\renewcommand\citepright{)}
%\renewcommand\citepform[1]{\textsl{#1}}

\usepackage{amsmath}

% Remove brackets from numbering in list of References
%\renewcommand\refname{\large References}
%\makeatletter
%\renewcommand{\@biblabel}[1]{\quad#1.}
%\makeatother

\usepackage{authblk}
\renewcommand\Authands{ \& }
\renewcommand\Authfont{\normalsize \bf}
\renewcommand\Affilfont{\small \normalfont}
\makeatletter
\renewcommand\AB@affilsepx{, \protect\Affilfont}
\makeatother

% comments
\usepackage{ulem}
\definecolor{purple}{rgb}{0.459,0.109,0.538}
\def\tb#1#2{\sout{#1} \textcolor{purple}{#2}} 
\def\tbc#1{\textcolor{purple}{[#1]}}

% symbols
\newcommand{\chiSq}{\chi^{2}_{df}}
\newcommand{\dtmrca}{\Delta_\mathrm{TMRCA}}
\newcommand{\dspr}{d_\mathrm{SPR}}

%%% TITLE %%%
\title{\vspace{1.0cm} \LARGE \bf Reassortment between influenza B lineages and the emergence of a co-adapted PB1-PB2-HA gene complex}

\author[1]{Gytis Dudas}
\author[2]{Trevor Bedford}
\author[1,3]{Samantha Lycett}
\author[1,4,5]{Andrew Rambaut}

\affil[1]{Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK}
\affil[2]{Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center, Seattle, WA, USA}
\affil[3]{Institute of Biodiversity Animal Health and Comparative Medicine, University of Glasgow, Glasgow, UK}
\affil[4]{Fogarty International Center, National Institutes of Health, Bethesda, MD, USA}
\affil[5]{Centre for Immunology, Infection and Evolution at the University of Edinburgh, Edinburgh, UK}

\date{\today}

\begin{document}
\maketitle

\begin{abstract}

Influenza B viruses are increasingly being recognized as major contributors to morbidity attributed to seasonal influenza. 
Currently circulating influenza B isolates are known to belong to two antigenically distinct lineages referred to as B/Victoria and B/Yamagata. 
Frequent exchange of genomic segments of these two lineages has been noted in the past, but the observed patterns of reassortment have not been formalized in detail.
We investigate inter-lineage reassortments by comparing phylogenetic trees across genomic segments.
Our analyses indicate that of the 8 segments of influenza B viruses only PB1, PB2 and HA segments maintained separate Victoria and Yamagata lineages and that currently circulating strains possess PB1, PB2 and HA segments derived entirely from one or the other lineage; other segments have repeatedly reassorted between lineages thereby reducing genetic diversity.
We argue that this difference between segments is due to selection against reassortant viruses with mixed lineage PB1, PB2 and HA segments.
Given sufficient time and continued recruitment to the reassortment-isolated PB1-PB2-HA gene complex, we expect influenza B viruses to eventually undergo sympatric speciation.

\end{abstract}

\pagebreak

\section*{Introduction}
Seasonal influenza causes between 250,000 and 500,000 deaths annually and comprises lineages from three virus types (A, B and C) co-circulating in humans, of which influenza A is considered to cause the majority of seasonal morbidity and mortality \citep{flufactsheet}.
However, influenza B viruses are increasingly being recognized as important human pathogens \citep{paul-glezen2013} and occasionally come to predominate an influenza season.
For example in the 2012/2013 European season as many as 53\% of influenza sentinel surveillance samples tested positive for influenza B \citep{ECDC1213}. 

Like other members of \textit{Orthomyxoviridae}, influenza B viruses have segmented genomes, which allow viruses co-infecting the same cell to exchange segments, a process known as reassortment. 
Influenza A viruses are widely considered to be a major threat to human health worldwide due to their ability to cause pandemics in humans via reassortment of circulating human strains with non-human influenza A strains. 
Although influenza B viruses have been observed to infect seals \citep{osterhaus2000,bodewes2013} through a reverse zoonosis, they are thought to primarily infect humans and are thus unlikely to exhibit pandemics due to the absence of an animal reservoir from which to acquire antigenic novelty. 
Both influenza A and B evolve antigenically through time in a process known as antigenic drift, in which mutations to the haemagglutinin (HA) protein allow viruses to escape existing human immunity and persist in the human population, leading to recurrent seasonal epidemics \citep{burnet1955,hay2001,bedford2014}.

Currently circulating influenza B viruses comprise two distinct lineages -- Victoria and Yamagata (referred to as Vic and Yam, respectively) -- named after strains B/Victoria/2/87 and B/Yamagata/16/88, that are thought to have genetically diverged in HA around 1983 \citep{rota1990}. 
These two lineages now possess antigenically distinct HA surface glycoproteins \citep{kanegae1990,rota1990,nerome1998,nakagawa2002,ansaldi2003} allowing them to co-circulate in the human population.
Phylogenetic analysis of evolutionary rate, selective pressures and reassortment history of influenza B has shown extensive and often complicated patterns of reassortment between all segments of influenza B viruses both between and within the Vic and Yam lineages \citep{chen2008}.

Here, we extend previous methods to reveal an evolutionarily intriguing pattern of reassortment in influenza B.
In our approach, membership to either the Victoria or Yamagata lineage in the tree of one segment is used to label the individual isolates in the tree of the other segments.
By modelling the transition between labels on a phylogenetic tree, reassortment events which result in the replacement of one segment's lineage by another show up as label changes along a branch (Figure \ref{methodFig}).
We use this method to reconstruct major reassortment events and quantify reassortment dynamics over time in a dataset of 452 influenza B genomes, and conduct secondary analyses in a dataset of 1603 influenza B genomes.

We show that despite extensive reassortment, three of the eight segments -- two segments coding for components of the influenza B virus polymerase, PB1 and PB1, and the surface glycoprotein HA -- still survive as distinct Victoria and Yamagata lineages, which appear to be co-adapted to the point where virions which do not contain PB1, PB2 or HA segments derived entirely from either the Vic or the Yam lineage have rarely been isolated and only circulate as transient lineages once isolated.
In other segments (PA, NP, NA, MP and NS) a single lineage has introgressed into the opposing background and been fixed in the influenza B population: Yam for PA, NP, NA and MP and Vic for NS.
This has occurred through repeated reassortments and subsequent fixation of reassortant genome constellations within the influenza B population.

\begin{figure}[h]
 \centering		
	\includegraphics[width=0.45\textwidth]{figures/TreeFigure2}
	\caption{\textbf{Schematic analysis of reassortment patterns.}
	A) We begin by assigning sequences falling on either side of a specified bifurcation within each segment tree to different lineages, in this case, the Victoria and Yamagata bifurcation that occurred in the early 1980s.
	B) We then transfer lineage labels from one tree to the same tips in another tree.
	Transitions between labels along this second tree thus indicate reassortment events that combine lineages falling on different sides of the Vic/Yam bifurcation in the first tree.
	C) A reassortment graph depiction shows that tip number 6 is determined to be a reassortant based on B).
	}
	\label{methodFig}
\end{figure}

\section*{Results}

\subsection*{Analysis of reassortment patterns across Victoria and Yamagata lineages}
The differentiation into Vic and Yam lineages can be seen in all segments (Figure \ref{genomeGrid}).
Following the split of the two lineages, each segment can be assigned to either Vic or Yam lineage and inter-lineage reassortment events have yielded mixed-lineage genome constellations.
On some segments, either the Victoria or Yamagata lineage fixed in the influenza B virus population, i.e.\ became the `trunk' of the phylogenetic tree of a segment, seen as modern viruses deriving completely from either the Victoria or Yamagata lineage (yellow vs purple bars in Figure \ref{genomeGrid}).
This pattern is apparent in the PA, NP, NA, MP and NS segments.
However, the PB1, PB2 and HA segments of modern viruses are derived from both Victoria and Yamagata lineages.
Consistent with fixation of Victoria or Yamagata lineages, the PA, NP, NA, MP and NS segments periodically lose diversity, while maintenance of parallel Victoria and Yamagata lineages results in continually increasing diversity in segments PB1, PB2 and HA (Figure \ref{tmrcaOT}).
The PB1, PB2 and HA segments from present-day viruses maintain a common ancestor in $\sim$1983 and thus accumulate genetic diversity since the split of those segments into Vic and Yam lineages, while other segments often lose diversity with ancestors to present-day viruses appearing between $\sim$1991 and $\sim$1999.

\begin{figure}[h]
\centering
\includegraphics[width=0.95\textwidth]{figures/InfB_genomeGrid.pdf}
\caption{\textbf{Maximum clade credibility (MCC) trees of all 8 genome segments of influenza B viruses isolated since 1980.}
Trees are coloured based on inferred PB1-PB2-HA lineage.
Vertical bars indicate the original Victoria and Yamagata lineages within each segment.
Each tree corresponds to the summarised output from single analyses comprised of 9000 trees sampled from the posterior distribution of trees.
}
\label{genomeGrid}
\end{figure}

\begin{figure}[h]
	\centering		
	\includegraphics[width=0.75\textwidth]{figures/InfB_tmrcaOT_lines.png}
	\caption{\textbf{Oldest TMRCA of all surviving branches over time.}
PA, NP, NA, MP and NS segments of influenza B viruses show periodic losses of diversity, indicating lineage turnover.
PB1, PB2 and HA segments, on the other hand, maintain the diversity dating back to the initial split of Vic and Yam lineages.
Each point is the mean time of most recent common ancestor (TMRCA) of all surviving lineages existing at each time slice through the tree and vertical lines indicating uncertainty are 95\% highest posterior densities (HPDs).}
	\label{tmrcaOT}
\end{figure}

By measuring mean pairwise diversity between branches in each tree that were assigned either a Vic or Yam label in other segments, we look for reductions in between-lineage diversity, which indicate that an inter-lineage reassortment event has taken place (Figure \ref{betweenDiversity}).
This method gives a quantitative measure of reassortment-induced loss of diversity between Victoria and Yamagata lineages in two trees, although care should be taken when interpreting the statistic, as it does not correspond to any real TMRCAs in the tree, but can be interpreted as mean coalescence date between Vic and Yam lineages of PB1, PB2 and HA segments in all other trees.
We focus only on PB1, PB2 and HA lineage labels, since all other segments have fixed either the Vic or the Yam lineage.
Losses of diversity (represented by more recent mean pairwise TMRCAs between Vic and Yam labels) in Figure \ref{betweenDiversity} indicate that every segment has reassorted with respect to the Victoria and Yamagata lineages of PB1, PB2 and HA segments.
However, we also see that the labels for these 3 segments show reciprocal preservation of diversity after 1997.
This suggests that after 1997 no reassortment events have taken place between Victoria and Yamagata lineages of PB1, PB2 and HA segments and their lineage labels only `meet' at the root.
We do see reduced diversity between Vic and Yam labels of PB1, PB2 and HA segments in a time period close to the initial split of Vic and Yam lineages (1986--1996).
These reductions in diversity represent small clades with reassortant PB1-PB2-HA constellations, which go extinct by 1997.
We also observe that the assignment of these 3 segment labels to branches of other segment trees is very similar and often identical after 1997.
This suggests that PB1, PB2 and HA lineage labels switch simultaneously in all trees after 1997.

\begin{figure}[h]
	\centering		
	\includegraphics[width=0.75\textwidth]{figures/InfB_betweenDiversity.png}
	\caption{\textbf{Mean pairwise TMRCA between Vic and Yam branches under PB1, PB2 and HA label sets.}
PB1, PB2 and HA segment labels indicate that these segments show reciprocal preservation of diversity, which dates back to the split of Vic and Yam lineages.
All other segments show increasingly more recent TMRCAs between branches labelled as Vic and Yam in PB1, PB2 and HA label sets.
All vertical lines indicating uncertainty are 95\% highest posterior densities (HPDs).}
	\label{betweenDiversity}
\end{figure}

We show the ratio of Vic and Yam lineages in our sequence data in different influenza seasons in Figure \ref{lineageRatiosOverTime}, which is based on lineage assignment performed earlier (see Methods).
It is evident that losses of diversity in the PA, NP, NA, MP and NS segments are related to the repeated fixation of either the Vic or the Yam lineage.
These losses of diversity correspond to fixation of the Vic lineage in NS and fixation of the Yam lineage in PA, NP, NA and MP.
Similarly, the lack of reassortment between Vic and Yam lineages and maintenance of diversity of PB1, PB2 and HA can be seen, where the two lineages have been isolated at a ratio close to 50\% over long periods of time (Figure \ref{lineageRatiosOverTime}).
On a year-to-year basis, however, the ratios for Vic and Yam lineage PB1, PB2 and HA can fluctuate dramatically consistent with one lineage predominating within a given season, in agreement with surveillance data \citep{reed2012}.

\begin{figure}[h]
	\centering	
	\includegraphics[width=0.65\textwidth]	{figures/InfB_LineageRatiosOverTime.png}
	\caption{\textbf{Ratio of lineages in the dataset.}
The ratio of sequences that derive from the original Victoria clade (yellow) to sequences that derive from the Yamagata clade (purple) in each segment over time.
Black lines indicate where this ratio lies in the larger 1600 genomes dataset.
Numbers at the top of the figure show the total number of genomes analyzed within each influenza season in the main dataset comprised of 452 genomes from which the ratio was calculated, while the numbers in brackets correspond to numbers of sequences in the larger 1600 genomes dataset.
Numbers at the bottom are influenza seasons from the 1987/1988 (87/88) season to the 2011/2012 season.
Yamagata lineage PA, NP, NA and MP segments and Victoria lineage NS segment eventually become fixed in the influenza B population.
PB1, PB2 and HA segments maintain separate Victoria and Yamagata lineages.}
	\label{lineageRatiosOverTime}
\end{figure}

We reconstructed reassortment events that were detected by using lineage labels.
Figure \ref{railroadPlot} focuses only on inter-lineage reassortments that have occurred after 1990.
We identify 5 major (in terms of persistence) reassortant genome constellations (given in order PB1-PB2-PA-HA-NP-NA-MP-NS with prime (') indicating independently acquired segments) circulating between 1992 and 2011 (Figure \ref{railroadPlot}):
\begin{itemize}
  \item B/Alaska/12/1996-like (Y-Y-Y-Y-Y-Y-Y-V)
  \item B/Nanchang/2/1997-like (V-V-Y-V-Y-V-Y-V)
  \item B/Iowa/03/2002-like (V-V-Y'-V-Y-Y-Y'-V')
  \item B/California/NHRC0001/2006-like (V-V-Y-V-Y'-Y-Y'-V')
  \item B/Brisbane/33/2008-like (V-V-Y-V-Y'-Y-Y-V)
\end{itemize}
B/Alaska/12/1996-like, B/Nanchang/2/1997-like and B/Iowa/03/2002-like constellations were observed by \citet{chen2008}, but sequences from \\B/California/NHRC0001/2006-like and B/Brisbane/33/2008-like constellations were not available at the time.
In their study \citet{chen2008} also recovered the co-assortment pattern of PB1, PB2 and HA lineages, but did not remark upon it.
Of these 5 constellations 4 (B/Nanchang/2/1997-like, B/Iowa/03/2002-like,\\ B/California/NHRC0001/2006-like and B/Brisbane/33/2008-like) are derived from introgression of Yamagata lineage segments into Victoria lineage PB1-PB2-HA background, with only 1 (B/Alaska/12/1996-like) resulting from introgression of Victoria lineage NS segment into an entirely Yamagata derived background.
All 5 inter-lineage reassortment events described here are marked by the preservation of either entirely Victoria or Yamagata derived PB1-PB2-HA segments.
Figure \ref{railroadPlot} also shows that reassorting segments appear to evolve with a considerable degree of autonomy.
For example, the NP lineage that entered a largely Victoria lineage derived genome and gave rise to the B/Nanchang/2/1997-like isolates continued circulating until 2010, even though other segments it reassorted with in 1995 -- 1996 (PA and MP) went extinct following the reassortments that led to the rise of viruses with B/Iowa/03/2002-like genome constellations.
A more extreme example is the NS segment, which in B/Iowa/03/2002-like isolates (and all subsequent Vic PB1-PB2-HA isolates) has originally been derived from the Victoria lineage that had been associated with mostly Yam lineage derived B/Alaska/12/1996-like genomes for a number of years.

We observe that in 5 successful inter-lineage reassortment events, none break up the PB1-PB2-HA complex.
This is an unlikely outcome -- the probability of not breaking up PB1-PB2-HA across 5 reassortment events is $p = (\frac{2^{5} \times 2 - 2}{2^{8} - 2})^{5} = 0.0009$, where reassortment events are considered to sample from the Vic and Yam lineages at random for each of the 8 segments.
If we correct for multiple testing with the assumption that co-assortment of any 3 segments is of interest we find that the probability of not breaking up an arbitrary 3 segments across 5 reassortment events is $p = \binom{8}{3} \times (\frac{2^{5} \times 2 - 2}{2^{8} - 2})^{5} = 0.0485$.

\begin{figure}[h]
	\centering		
	\includegraphics[width=0.95\textwidth]{figures/RailroadPlotDated.pdf}
	\caption{\textbf{Schematic plot of reconstructed reassortments between Victoria and Yamagata lineage segments of influenza B virus.}
Lineages that coassort in genomes are represented by 8 parallel lines, with lineages that derive from the original Victoria clade colored yellow/brown and lineages that derive from the original Yamagata clade colored lilac/purple.
Inter-lineage reassortment events are indicated by lines entering a different genome.
The angle of incoming lineages represents uncertainty in the timing of the event (mean date of the reassortant node and its parent node).
Lineage extinction dates are not shown accurately.}
	\label{railroadPlot}
\end{figure}

Although the vast majority of influenza B isolates possess either Vic or Yam lineage derived PB1-PB2-HA complexes, on rare occasions mixed-lineage PB1-PB2-HA constellations emerge.
Figure \ref{stateTime} shows the sum of branch lengths which were labelled as having entirely Vic, entirely Yam or mixed-lineage PB1, PB2 and HA segments.
Due to lack of reassortment between Vic and Yam lineages of PB1, PB2 and HA (Figure \ref{betweenDiversity}) since 1997 all segments have spent significantly longer periods of evolutionary time with either entirely Vic-derived or entirely Yam-derived than with mixed-lineage PB1, PB2 and HA constellations (Figure \ref{stateTime}).
We have identified 3 instances of mixed-lineage PB1-PB2-HA reassortants from the data with the following PB1-PB2-HA constellations: VVY (B/Bangkok/163/1990-like, 13 sequences isolated 1990 -- 5 Jan 1995), VYV (B/Nanchang/630/1994-like, 2 sequences isolated 1994 -- 1996) and VYY (B/New York/24/1993-like, 2 sequences isolated 8 Jan 1993 -- 1994).
We detected two new reassortant lineages when investigating the larger (1600 genomes) dataset -- B/Waikato/6/2005-like viruses with PB1-PB2-HA constellation YYV (17 sequences isolated 9 May -- 12 October in 2005) and B/Malaysia/1829782/2007 with PB1-PB2-HA constellation YVY (1 sequence isolated 2 August 2007).

\begin{figure}[h]
	\centering		
	\includegraphics[width=0.65\textwidth]{figures/InfB_stateTimeAlt.png}
	\caption{\textbf{Amount of evolutionary time each segment has spent under different PB1-PB2-HA constellations.}
All segments have spent significantly more of their history with entirely Vic or entirely Yam-derived PB1-PB2-HA complexes.
All vertical lines indicating uncertainty are 95\% highest posterior densities (HPDs).}
	\label{stateTime}
\end{figure}

\subsection*{Analysis of reassortment properties}
We attempted to quantify the temporal discordance between lineages reassorting into new genomic constellations.
If one were able to recover an influenza `species tree', including admixture/reassortment events, it would be possible to estimate the reassortment or recombination `distance', which is the time between a split in the species tree in the past and a reassortment event (see Figure \ref{speciesTree}).
Although we do not find evidence of differences in numbers of overall reassortments between segments (see Supplementary information), we find support for a reassortment `distance' effect, in which a pair of tips on one segment has a different TMRCA from the same pair of tips on a different segment.
This difference in TMRCAs, or $\dtmrca$, is most sensitive when only one of the two trees being compared loses diversity via reassortment and the other acts like a proxy for the `species tree'.
We normalize our $\dtmrca$ comparisons to account for alignment length induced uncertainty in tree topology stability over the course of the MCMC chain (see Methods).
Figure \ref{deltaTMRCA} shows normalized mean $\dtmrca$ values for all pairs of trees.
Most segment pairs show very low values for this statistic with $\dtmrca \approx 0.1$, indicating that $\dtmrca$ measurements between replicate posterior samples from the same segment are up to 10 times smaller than $\dtmrca$ values between posterior samples from different segments.
PB1, PB2 and HA trees, on the other hand, exhibit normalized $\dtmrca$ values that are much higher.
This shows that TMRCA differences between trees of PB1, PB2 and HA segments are, though noisy, occasionally very similar to uncertainty in tip-to-tip TMRCAs between replicate analyses of these segments.

\begin{figure}[h]
	\centering		
	\includegraphics[width=0.65\textwidth]{figures/InfB_normalizedMuDeltaTMRCA.png}
	\caption{\textbf{Normalized mean $\dtmrca$ statistics between pairs of segments.}
PB1, PB2 and HA trees exhibit reciprocally highly similar TMRCAs, unlike most other pairwise comparisons.
All vertical lines indicating uncertainty are 95\% highest posterior densities (HPDs).}
	\label{deltaTMRCA}
\end{figure}

Linkage disequilibrium (LD) is a measure of non-random association between pairs of alleles at different polymorphic loci within a population.
To estimate LD no other information, except for haplotype and allele frequencies at polymorphic amino acid sites, is required.
Although not an external validation of previous phylogenetic results, we observe greater LD values between PB1, PB2 and HA than between other pairs of segments (Figure \ref{segmentLD}) in a large (approximately 1600 genomes) dataset.
This suggests that PB1, PB2 and HA segments possess a considerable number of co-assorting alleles, which upon closer inspection are associated with either Vic or Yam lineage segments.
We conclude that Victoria and Yamagata lineages of PB1, PB2 and HA have accumulated lineage-specific amino acid substitutions.
Of the amino acid sites that exhibit high LD on PB1, PB2 and HA proteins, there are 4 sites on PB1, 4 on PB2 and 4 on HA proteins which form a network of sites exhibiting high LD (Figures \ref{ChiGenome} and \ref{ChiCore}).
These sites define the split between Vic and Yam lineages within PB1, PB2 and HA segments.
In addition, there are sites on PB1, PB2, HA and NA proteins which also show high, albeit smaller, LD which correspond to sites which have undergone amino acid replacements some time after the Vic/Yam split.

\begin{figure}[h]
	\centering	
	\includegraphics[width=0.65\textwidth]{supp_figures/InfB_LDcircle.1600.aa.ChiSqdf.minorCutoff1percent.png}
	\caption{\textbf{LD comparison between influenza B proteins.}
Pairwise comparisons of linkage disequilibrium between amino acid sites on influenza B proteins.
Many polymorphic amino acid sites on PB1, PB2 and HA proteins exhibit high LD between themselves, followed by the NA protein.
This is evidence of a considerable number of co-assorting alleles within these proteins.}
	\label{segmentLD}
\end{figure}

\section*{Discussion}

\subsection*{Evidence of a co-adapted PB1-PB2-HA gene complex}
In this paper we show that the PB1, PB2 and HA segments of influenza B viruses are the only ones that have continuously maintained separate Vic and Yam lineages, while other segments have fixed either Vic or Yam lineages (Figures \ref{genomeGrid}, \ref{lineageRatiosOverTime} and \ref{railroadPlot}).
Evidence suggests that this is a result of prolonged lack of reassortment between Vic and Yam lineages in PB1, PB2 and HA (Figure \ref{betweenDiversity}) which possess co-assorting sequences detectable as high linkage disequilibrium (Figure \ref{segmentLD}).
The vast majority of the sampled evolutionary history of each segment of influenza B viruses since the split of Vic and Yam lineages has been spent in association with either completely Victoria or completely Yamagata lineage derived PB1-PB2-HA complexes (Figure \ref{stateTime}), suggesting that having `pure' lineage PB1-PB2-HA complexes is important for whole-genome fitness.
We propose that this pattern of coassortment is due to the action of selection and not simply biased or rare reassortment.

When comparing the mean tip-tip TMRCA deviations we find evidence of isolation by temporal distance - when reassortments occur between PB1, PB2 and HA segments those events tend to involve branches that have similar TMRCAs.
Other segment pairs, with the exception of MP-NA, seem to show little signal of this and are able to reassort with branches that have been evolving under temporally distant genomic backgrounds.
In addition, most strains with mixed-lineage PB1-PB2-HA complexes occurred in the early years of the Vic--Yam split, when the two lineages were presumably much more similar at the nucleotide and amino acid levels.
The most recent influenza B viruses with mixed-lineage PB1-PB2-HA complexes are B/Waikato/6/2005-like and B/Malaysia/1829782/2007-like viruses which circulated for only 5 months or are known from a single isolate, respectively.

We should clarify that when we say that Vic and Yam PB1-PB2-HA complexes of the influenza B virus are co-adapted and have co-evolved we mean that these 3 segments do not appear to function well when derived from both lineages and that the evolution to this state has been intimate, in the sense that there has not been a successful reassortant combining these two lineages across great temporal distances, unlike the rest of the genome.


\subsection*{Causes of co-dependence between PB1-PB2-HA}
Previous studies have investigated possible co-dependence patterns between segments of influenza B viruses, by focusing on segments which would be expected to be co-adapted, \textit{e.g.} PB1-PB2-PA and HA-NA \citep{mccullers2004}.
Though it would be easy to explain co-adaptation between these segments by referring to their functional roles \textit{e.g.} PB1-PB2-PA form the polymerase heterotrimer and HA-NA have antagonistic activities, our findings suggest a counter-intuitive relationship between PB1, PB2 and HA segments.

It is perhaps easiest to explain co-dependence between PB1 and PB2 segments based on their functions as part of the influenza RNA-dependent RNA polymerase (RdRp) heterotrimer.
Indeed, trees of PB1 and PB2 segments exhibit high similarity in tip--tip TMRCAs (Figure \ref{deltaTMRCA}), suggesting highly similar TMRCAs.
In addition, PB1--PB2 reassortants are the rarest and least persistent among mixed-lineage PB1-PB2-HA strains and have not been isolated in great numbers.

Explaining the co-dependence of PB1+2 and HA segments is more difficult.
Many studies have noted a possible link between the HA, NA and PB1 segments of influenza A viruses \citep{bergeron2010,fulvini2011}.
A previously used technique for producing vaccine seed strains involved selecting for HA-NA reassortants, which often yielded PB1-HA-NA reassortants as a side-effect \citep{bergeron2010,fulvini2011}.
Recent experiments have suggested that the presence or absence of a `foreign' PB1 segment can have dramatic effects on HA concentration on the surface of virions and total virion production \citep{cobbin2013}.
Additional evidence for a relationship between PB1 and HA segments in influenza A viruses is given by previous influenza pandemics, which were caused by avian-human influenza A virus reassortants.
It has been established that at least for the 1957 and the 1968 influenza pandemics, caused by A/H2N2 and A/H3N2 subtypes, respectively, the viruses responsible were reassortants possessing PB1 and HA segments derived from avian influenza A viruses \citep{kawaoka1989}.
However, there have also been reassortant influenza A viruses circulating for prolonged periods of time in humans that did have disparate PB1 and HA segments, \textit{e.g.} H1N2 outbreaks in 2001 \citep{gregory2002} and H1N1/09 in 2009 \citep{smith2009}.

Another possibility is the action of balancing selection in preserving the diversity in one segment, whilst the other segments hitchhike along.
A good candidate for this would be HA, as it is now the sole bearer of substantial antigenic diversity within the influenza B population, as the Vic lineage NA segment went extinct in 2002 (Figures \ref{genomeGrid}, \ref{tmrcaOT}, \ref{lineageRatiosOverTime} and \ref{railroadPlot}).
Previous research has found that avian influenza A virus HA and NA segments, which are the primary vehicles of antigenicity, exhibit vast diversity when compared to `internal segments' (including PB1 and PB2 segments), which show much more recent TMRCAs and less variability at the amino acid level \citep{chen2006,obenauer2006}.
This is easiest to interpret as frequency-dependent balancing selection acting to preserve antigenic diversity \citep{worobey2014}.
If this is the case in influenza B viruses, we expect balancing selection to act on HA and indirectly, through some unknown association with HA, on PB1 and PB2 segments.

We find this hitchhiking scenario unlikely: if PB1+2 segments were hitchhiking with the HA segment stochastically we would expect to see more PB1+2 versus HA reassortants and fixation of Vic or Yam lineage PB1+2 segments in the influenza B population.
Our main dataset comprising 452 complete influenza B virus genomes has one instance of these kinds of reassortants characterised by B/Bangkok/163/1990-like genome constellations.
The larger dataset with 1600 complete genomes has one additional example of this: isolates with a B/Waikato/6/2005-like PB1-PB2-HA constellation.
We note that in all cases the genome constellations of these reassortants were not fixed in the influenza B population and the more recent B/Waikato/6/2005-like reassortants persisted for a much shorter period of time (5 months versus 4 years), suggestive of declining PB1+2/HA reassortant fitness over time.

It is possible that Vic and Yam lineages of PB1, PB2 and HA segments have simply drifted away from each other, without any one segment being the driver of diversity preservation in PB1, PB2 and HA segments.
In this case PB1, PB2 and HA segments of Vic and Yam lineage accumulate substitutions that improve their ability to co-operate with segments of the same lineage and worse at interacting segments of a different lineage.
This process, termed mutation-driven co-evolution \citep{presgraves2010}, has been suggested to be the cause of hybrid dysfunction in \textit{Saccharomyces} hybrids \citep{lee2008}.
It is widely accepted that Victoria lineage HA had been restricted to eastern Asia between 1992 and 2000 \citep{nerome1998,shaw2002}, offering a potential explanation for why the budding Victoria lineage segments were not homogenized via reassortment with Yamagata lineage segment in the early years of the split between the two lineages. 
The restriction to eastern Asia would presumably also give sufficient time for PB1, PB2 and HA segments to co-evolve together.
Similarly to the previous scenario, however, mutation-driven co-evolution would require an association of some kind between PB1+2 and HA segments to explain the low observed frequency and poor fitness of PB1+2/HA reassortants.
In addition, we see no reason why these three particular segments, and not the rest of the genome, would become co-adapted to each other in geographic isolation.

\subsection*{Suggested experiments}
The association between Victoria and Yamagata lineage PB1-PB2-HA complexes should be relatively straightforward to test in the lab.
Using previously developed plasmid systems \citep{hoffmann2002} it would be possible to create artificial reassortants, combining Vic and Yam lineages of PB1, PB2 and HA segments into mixed-lineage PB1-PB2-HA complexes.
We predict that artificially produced viruses with mixed-lineage PB1-PB2-HA complexes will have reduced fitness when compared to viruses with pure-lineage, i.e.\ entirely Vic or entirely Yam, PB1-PB2-HA complexes.
In addition, we expect the relationship between Vic and Yam lineage PB1, PB2 and HA segments to be dependent on date of segment isolation, as viruses with mixed-lineage PB1-PB2-HA complexes isolated earlier should perform better than viruses with PB1-PB2-HA segments isolated more recently.

Given the short circulation times of recent PB1-PB2-HA reassortants B/Waikato/6/2005-like and B/Malaysia/1829782/2007-like viruses we expect that artificially produced PB1-PB2 reassortants would be much less fit than either PB1-HA or PB2-HA reassortants.
We thus expect the following hierarchy of reassortant fitnesses (in order of decreasing fitness): PB1-PB2-HA, PB1-PB2/HA and PB1-HA/PB2 or PB2-HA/PB1, though there is some evidence to suggest that PB1-HA/PB2 might be more fit than PB2-HA/PB1 (Figures \ref{NormSPR_PB1-PB2_difference} -- \ref{NormSPR_PB2-HA_difference}).
The history of reassortments in influenza B viruses (Figure \ref{railroadPlot}) suggests that there are lineage-specific effects too, given the almost universal introgression of Yamagata lineage segments into Victoria PB1-PB2-HA background.
However, our analyses do not indicate any obvious differences in synonymous, non-synonymous or nucleotide substitution rates between Vic and Yam PB1-PB2-HA segment complexes or segments associated with either of the two (Figure \ref{robustCounting}).

However, we also see that epistatic effects might interfere with fitness measurements, if for example non-PB1-PB2-HA segments are also temporally mismatched.
Ideally, the co-adaptation would be easier to understand by referring to the structures of PB1 and PB2 proteins, as the link between these would be intuitive.
We have identified amino acid sites which are linked between PB1, PB2 and HA proteins of Victoria and Yamagata lineages, but we find very few sites on PB1 and PB2 proteins (Figure \ref{ChiCore}) that fall within the regions that form contacts within the influenza B polymerase heterotrimer \citep{sugiyama2009}, suggesting more subtle roles for sites we have identified.

\subsection*{The future of influenza B viruses}
We suggest that the preservation of two PB1-PB2-HA complex lineages is similar to genomic speciation islands, where small numbers of genes resist being homogenized through gene flow \citep{turner2005}.
In this context, we see three potential paths of evolution for influenza B viruses.
More segments could be recruited into the two currently circulating co-adapted segment complexes (PB1, PB2 and HA segments being the genomic speciation islands), as part of a speciation process, until all circulating influenza B viruses possess genomes with segments firmly associated with either the Vic or Yam lineage PB1-PB2-HA complex which could be referred to as belonging to either `new Victoria' or `new Yamagata' lineages.
This is the speciation scenario.
Due to the rarity of inter-lineage reassortment events it is unclear whether this process is already under way with NA and MP, based on Figures \ref{genomeGrid}, \ref{betweenDiversity} and \ref{deltaTMRCA}, being the next segments to be recruited to PB1-PB2-HA complexes.
The two alternatives to the speciation hypothesis are the `\textit{status quo}' model, where the influenza B genome continues to be homogenized via gene flow with the exception of PB1, PB2 and HA segments and the extinction scenario, whereby one of the two PB1-PB2-HA complexes goes extinct, marking the return of single-strain dynamics in the influenza B virus population.

Sympatric speciation in other systems usually requires strong barriers to introgression, \textit{e.g.} infertility of F1 hybrids can lead to the evolution of prezygotic reproductive isolation otherwise known as reinforcement or the Wallace effect.
To what extent this would apply to influenza B viruses remains unknown.
If influenza B viruses are undergoing sympatric speciation, it is imperative to determine whether co-infection with Victoria and Yamagata lineages of PB1-PB2-HA segments occur at a sufficiently high frequency and result in considerable losses of fitness to drive the evolution of reassortment isolation mechanisms (\textit{e.g.} unique packaging signals) or whether co-infection is so rare that speciation occurs via mutation-driven co-evolution.
We think that co-dependence between PB1 and PB2 segments can be explained by the fact that they are functionally linked: together with the PA protein they form part of the influenza B virus RNA-dependent RNA polymerase heterotrimer.

It also remains unknown whether reassortment events of the past 20 years (Figure \ref{railroadPlot}) which frequently involved the NP, MP and NS segments from a Yamagata PB1-PB2-HA background reassorting into a Victoria PB1-PB2-HA background are indicative of rare events followed by selective sweeps or stochastic fixation.
It's not unfeasible for selective sweeps following reassortments to break down developing co-adaptation of segments, especially if they are not functionally linked and have themselves been reassorted into a new PB1-PB2-HA background recently.
This is the second scenario we might expect to occur, whereby relatively frequent reassortments occur between Vic and Yam lineages and are followed by selective sweeps.
In this case influenza B viruses would undergo periodic genome homogenization events with the exception of PB1, PB2 and HA segments.
Because Vic HA and Yam HA are antigenically dissimilar we think that balancing selection would prevent even strong selective sweeps from driving the opposing PB1-PB2-HA gene complex to extinction.
This `\textit{status quo}' model would require strong selective sweeps and/or relatively frequent reassortments, neither of which seem to be lacking in the influenza B virus population.

Given the relatively recent explosion of sequence data available for influenza B, it is difficult to say whether dynamics similar to Victoria and Yamagata lineages have not occurred in influenza B virus genomes before and left no trace through extinction.
We find it unlikely that either Victoria or Yamagata lineage PB1-PB2-HA complexes will go extinct stochastically in the near future, as they have co-circulated for prolonged periods at a ratio close to 0.5, suggesting the action of balancing selection (Figure \ref{lineageRatiosOverTime}).
Extinction through depletion of susceptible individuals, such as following influenza pandemics or mass vaccination seem unlikely as well.
Both Victoria and Yamagata lineage PB1-PB2-HA complexes survived the admittedly mild influenza pandemic in 2009 and influenza vaccines, which are usually applied to specific subsets of the population, do not produce lifelong immunity.

\subsection*{Conclusion}
We have used the $\dtmrca$ statistic to determine the degree of similarity in TMRCA dates between two temporally calibrated phylogenies.
We believe that patristic distance methods such as this, though themselves far from being new, have considerable power to address a wide variety of problems when combined with temporal phylogenies.
One of many useful applications of the $\dtmrca$ method would be identifying clades or taxa that are products of reticulate evolution.
$\dtmrca$ measures between independent analyses of the same alignment (`within-alignment') could be used as a cutoff to detect outlier taxa with greater than expected `between-alignment' $\dtmrca$ values.
To develop further, however, the statistical properties of patristic distance methods have to be evaluated in greater detail.
In addition, by treating the relative position of each isolate within a phylogeny of one segment as a label and modelling it on phylogenies of other segments we have also developed a metric similar to `between population' diversity used in calculating $F_{ST}$, which is capable of quantifying reticulate evolution-induced loss of diversity between partitions of taxa in two or more phylogenies.

In this paper we apply a novel combination of population genetics and phylogenetic methods to full genome sequences in order to describe and quantify reassortment patterns in influenza B viruses circulating in humans from 1980 to the present day.
Our main finding is that in influenza B viruses only PB1, PB2 and HA segments maintain both Victoria and Yamagata lineages which associate with segments of their own lineage, yielding two co-circulating PB1-PB2-HA complexes: one entirely derived from Victoria and one from Yamagata lineage segments.
We argue that this is due to selection against viruses with mixed-lineage PB1-PB2-HA complexes.
Given sufficient time it should become clear whether PB1-PB2-HA complexes of human influenza B viruses are simply resisting gene flow whilst the rest of the influenza B virus genome is repeatedly homogenized or whether the two PB1-PB2-HA complexes are recruiting the rest of the genomic segments into co-adapted and co-reassorting segment complexes on their way to sympatric speciation.

\newpage

\section*{Methods}
We compiled a dataset of 452 complete influenza B genomes from GISAID \citep{GISAID} dating from 1984 to 2012 (accession numbers and laboratory acknowledgements can be found in Supplementary information).
The longest protein coding region of each segment was extracted and used for all further analyses.
We thus assume that homologous recombination has not taken place and that the evolutionary history of the whole segment can be inferred from the longest coding sequence in the segment.
To date there has been little evidence of homologous recombination in influenza viruses \citep{chare2003,boni2008,han2010}.
The segments of each strain were assigned to either Vic or Yam lineage by making maximum likelihood trees of each segment using PhyML \citep{guindon2003} and identifying whether the isolate was more closely related to B/Victoria/2/87 or B/Yamagata/16/88 sequences in that segment, with the exception of the NS segment (B/Victoria/2/87 was a reassortant and possessed a Yam lineage NS \citep{lindstrom1999}), where B/Czechoslovakia/69/1990 was considered as being representative of Victoria lineage.
Each strain was thus assigned 8 lineages depending on the combination of lineages from which their genomes were derived, for example all segments except for NS in strain B/Victoria/2/87 belong to Vic lineage and can thus be represented as (V,V,V,V,V,V,V,Y).

We also collated all complete influenza B virus genomes available on Genbank as of May 5, 2014.
After removing isolates that had considerable portions of any sequence missing, were isolated prior to 1980 or were suspected of having a contaminant sequence in any segment, we were left with 1603 sequences.
This dataset only became available after all primary analyses were performed, are mainly from Australia, New Zealand and the United States and are too numerous to analyze in BEAST \citep{drummond2012}.
PhyML \citep{guindon2003} was used to produce phylogenies of each segment and the lineage of each isolate was determined based on grouping with either B/Victoria/2/87 or B/Yamagata/16/88 sequences, as described above.
By associating strains with lineage identity of each of their segments, we reconstructed the most parsimonious inter-lineage reassortment history.

Temporally-calibrated phylogenies were recovered for each segment using Markov chain Monte Carlo (MCMC) methods in the BEAST software package \citep{drummond2012}.
Here, we modeled the substitution process using the HKY model of nucleotide substitution \citep{hky1985}, with separate transition models for each of the 3 codon partitions, and additionally estimate realized synonymous and non-synonymous substitution counts \citep{obrien2009}.
We used a flexible Bayesian skyride demographic model \citep{minin2008}.
We accounted for incomplete sampling dates for 94 sequences (of which 93 had only year and 1 had only year and month of isolation) whereby tip date is estimated as a latent variable in the MCMC integration.
A relaxed molecular clock was used, where branch lengths are drawn from a lognormal distribution \citep{drummond2006}.
We ran 3 independent MCMC chains, each with 200 million states, sampled every 20,000 steps and discarded the first 10\% of the MCMC states as burn-in.
After assessing convergence of all 3 MCMC chains by visual inspection using Tracer \citep{tracer}, we combined samples across chains to give a total of 27,000 samples from the posterior distribution of trees.

Every sequence was assigned 7 discrete traits in BEAUti corresponding to the lineages of all other segments with which a strain was isolated \textit{e.g.} PB1 tree had PB2, PA, HA, NP, NA, MP and NS as traits and V or Y as values for each trait.
We inferred the ancestral state of lineages in each segment by modelling transitions between these discrete states using an asymmetric transition matrix \citep{lemey2009} with Bayesian stochastic search variable selection (BSSVS) to estimate significant rates.
Because the posterior set of trees for a single segment has branches labelled with the inferred lineage in the remaining 7 segments, we can detect inter-lineage reassortments between pairs of segments by observing state transitions, i.e.\ Yam to Vic or Vic to Yam (Figure \ref{methodFig}).
In addition, by reconstructing the ancestral state of all other genomic segments jointly we can infer co-reassortment events when more than one trait transition occurs on the same node in a tree.

\subsection*{Measures of diversity}
We inferred the diversity of each segment at a single point in time by estimating the date of the most recent common ancestor of all branches at yearly intervals, which places an upper bound on the maximum amount of diversity existing at each time point.
A version of this lineage turnover metric has previously been used to investigate the tempo and strength of selection in influenza A viruses during seasonal circulation \citep{bedford2011}.
In addition, we calculated mean pairwise time of most recent common ancestor (TMRCA) between branches labelled as Vic and Yam for PB1, PB2 and HA traits.
This gave us a measure of how much a particular segment reassorts with respect to Vic and Yam lineages of PB1, PB2 and HA segments.
If Vic and Yam lineages of PB1, PB2 and HA segments were to be considered as being separate populations this measure would be equivalent to `between population' diversity.

We also calculated the total amount of evolutionary time spent by each segment with entirely Vic, entirely Yam or mixed lineage PB1, PB2 and HA segments.
We do this by summing the branch lengths in each tree under 3 different lineage combinations of the PB1, PB2 and HA segments: PB1-PB2-HA derived entirely from Yamagata lineage, PB1-PB2-HA entirely derived from Victoria lineage and PB1-PB2-HA derived from a mixture of the two lineages.
This gives a measure of how successful, over long periods of time, each particular PB1-PB2-HA constellation has been.

\subsection*{Tree to tree similarities}
We express the distance $\dtmrca$ between trees belonging to two segments $A$ and $B$ by averaging distances over posterior samples $(1, \dots, L)$, so that
\begin{equation}
	\dtmrca(A, B) = \frac{1}{L}\sum_{i=1}^L \delta(A_i, B_i).
\end{equation}
We calculate $\delta$ for an individual posterior sample following
\begin{equation}
\delta(A_i, B_i) = \frac{f(A_i, A_i') + f(B_i, B_i')}{2 \, f(A_i, B_i)},
\end{equation}
where $f(A_i, B_i) = \frac{1}{n}\sum_{j=1}^n g(A_{ij}, B_{ij})$ and $n$ is the total number of pairwise comparisons available between sets of tips.
Thus, $g(A_{ij},B_{ij})$ is the absolute difference in TMRCA of a pair of tips $j$, where the pair is drawn from the \textit{i}th posterior sample of tree $A$ and the \textit{i}th posterior sample of tree $B$.
Additionally, $f(A_i,A'_i)$ is calculated from the \textit{i}th posterior sample of tree A and \textit{i}th posterior sample of an independent analysis of tree $A$ (which we refer to as $A'$), to control for variability in tree topology stability over the course of the MCMC chain.
We had 3 replicate analyses of each segment and in order to calculate $f(A_i,A'_i)$ we used analyses numbered 1, 2 and 3 as $A$ and analyses numbered 2, 3 and 1 as $A'$, in that order.
We subsampled our combined posterior distribution of trees to give a total of 2700 trees on which to analyze $\dtmrca$.

\begin{figure}[h]
	\centering	
	\includegraphics[width=0.85\textwidth]{supp_figures/InfB_methodSpeciesTree.pdf}
	\caption{\textbf{Calculating $\dtmrca$ from a species tree perspective.}
Consider an organism that has diverged into 3 taxa (A, B, C) with a genome comprised of 3 segments (blue, green and red).
Due to reassortments taxa A and B share a slightly more recent TMRCA in the red segment, likewise for taxa B and C in the green segment.
By comparing differences in TMRCAs between taxa A-B, A-C and B-C in blue, red and green segments we would find that the red segment has a lower `reassortment distance' (X) than the green segment (Y).
}
	\label{speciesTree}
\end{figure}

Calculating $\dtmrca(A_i, B_i)$ for each MCMC state provides us with a posterior distribution of this statistic allowing  specific hypotheses regarding similarities between the trees of different segments to be tested.
Our approach exploits the branch scaling used by BEAST \citep{drummond2012}, since the trees are scaled in absolute time and insensitive to variation in nucleotide substitution rates between segments, allowing for direct comparisons between TMRCAs in different trees.
This method operates under the assumption that the segment trees capture the overall `species tree' (or `virus tree') of influenza B viruses quite well.
It is not an unreasonable assumption, given the seasonal bottlenecks influenza viruses experience.
This makes it almost certain that influenza viruses circulating at any given time point are derived from a single genome that existed in the recent past.
The $\dtmrca$ statistic essentially captures a facet of this influenza B `species tree' by quantifying the temporal distance between an admixture event and the original lineage that was replaced by the incoming lineage (see Figure \ref{speciesTree}).
Our $\dtmrca$ statistic is an extension of patristic distance methods and has previously been used to tackle a wide variety of problems, as phylogenetic distance in predicting viral titer in \textit{Drosophila} infected with viruses from closely related species \citep{longdon2011} and to assess temporal incongruence in a phylogenetic tree of amphibian species induced by using different calibrations \citep{ruane2011}.

\subsection*{Linkage disequilibrium across the influenza B genome}
We used the expanded Genbank dataset with approximately 1600 complete genome sequences to estimate linkage disequilibrium (LD) between amino acid loci across the longest proteins encoded by each segment of the influenza B virus genome.
To quantify LD we adapt the $\chiSq$ statistic from \citep{hedrick1986}:

\begin{equation}
\chiSq=\frac{\chi^{2}}{N\,(k-1)\,(m-1)},
\end{equation}

where $\chi^{2}$ is calculated from a classical contingency table, $N$ is the number of haplotypes and $(k-1)(m-1)$ are the degrees of freedom.
This statistic is equal to the widely used $r^2$ LD statistic at biallelic loci, but also quantifies LD when there are more than two alleles per locus \citep{zhao2005}.
LD was estimated only at loci where each nucleotide or amino acid allele was present in at least two isolates.
We ignored gaps in the alignment and did not consider them as polymorphisms.
In all cases we used a minor allele frequency cutoff of 1\%.
We also calculated another LD statistic, $D'$ \citep{lewontin1964}:

\begin{equation}
D'_{ij}=\frac{D_{ij}}{D_{ij}^{max}},
\end{equation}

where

\begin{equation}
D_{ij}=p(A_i B_j) - p(A_i) p(B_j)
\end{equation}

and

\begin{equation}
\begin{split}
D_{ij}^{max}=min[ p(A_i) p(B_j) , (1-p(A_i))(1-p(B_j)) ] \mbox{ when } D_{ij} < 0\\
D_{ij}^{max}=min[ (1-p(A_i)) p(B_j) , p(A_i)(1-p(B_j)) ] \mbox{ when } D_{ij} \geq 0,
\end{split}
\end{equation}

where $p(A_i)$ is the frequency of allele $A_i$ at locus A, $p(B_j)$ is the frequency of allele $B_j$ at locus B and $p(A_i B_j)$ is the frequency of haplotype $A_i B_j$.
$D'$ is inflated when some haplotypes are not observed \textit{e.g.} when the minor allele frequency is low.
We find that $D'$ is almost uniformly high across the influenza B virus genome and close to 1.0 for almost any pair of polymorphic loci.
This is because most amino acid alleles in the dataset exist transiently, meaning that they don't get a chance to reassort and we only observe them within the backgrounds of more persistent alleles, which $D'$ quantifies as complete LD.
We think that metrics related to $r^2$, like $\chiSq$, perform much better on temporal data such as ours in finding persistent associations between alleles and are easier to interpret.

\subsection*{Data availability}
Python scripts used to process trees and sequences, as well as their output are publicly available at:
\\https://github.com/evogytis/fluB/tree/master/data.

\section*{Acknowledgements}
We would like to thank Darren Obbard and Paul Wikramaratna for helpful discussions.
GD was supported by a Natural Environment Research Council studentship D76739X.
TB was supported by a Newton International Fellowship from the Royal Society. 
The research leading to these results has received funding from the European Research Council under the European Community's Seventh Framework Programme (FP7/2007-2013) under Grant Agreement no. 278433-PREDEMICS and ERC Grant agreement no. 260864.
AR and SL acknowledge the support of the Wellcome Trust (grant no. 092807).

\clearpage

%%% TITLE %%%
\title{\vspace{1.0cm} \huge \bf Supplemental information:\\ \LARGE Reassortment between influenza B lineages and the emergence of a co-adapted PB1-PB2-HA gene complex}

\author[1]{Gytis Dudas}
\author[2]{Trevor Bedford}
\author[1,3]{Samantha Lycett}
\author[1,4,5]{Andrew Rambaut}

\affil[1]{Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK}
\affil[2]{Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center, Seattle, WA, USA}
\affil[3]{Institute of Biodiversity Animal Health and Comparative Medicine, University of Glasgow, Glasgow, UK}
\affil[4]{Fogarty International Center, National Institutes of Health, Bethesda, MD, USA}
\affil[5]{Centre for Immunology, Infection and Evolution at the University of Edinburgh, Edinburgh, UK}

\date{\today}

\maketitle

\setcounter{figure}{0}
\setcounter{table}{0}
\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thetable}{S\arabic{table}}

\subsection*{Analysis of within-lineage reassortment patterns}
Subtree prune and regraft (SPR) distances between phylogenetic trees are an approximate measure of the numbers of reassortment or recombination events \citep{svinti2013}.
Exact SPR distances are difficult to compute, as they depend on the SPR distance itself and are impractical to compute for posterior distributions of trees except for the most similar trees.
We calculated approximate SPR distances \citep{whidden2009,whidden2010,whidden2013} to quantify the numbers of reassortments that have taken place between all pairs of segments.
Approximate SPR distances were normalized using the procedure used to normalize $\dspr$ (see Methods):
\begin{equation}
\dspr(A_i, B_i) = \frac{f(A_i, A_i') + f(B_i, B_i')}{2 \, f(A_i, B_i)},
\end{equation}
where $f(A_i, A_i')$, $f(B_i, B_i')$ and $f(A_i, B_i)$ are approximate SPR distances between \textit{i}th posterior samples from segments $A$, $B$ and independent analyses thereof ($A'$ and $B'$).
Figure \ref{SPRdistances} shows approximate SPR distances between all pairs of segment trees after normalization.
If there are biases in the way segments reassort, so that some segments tend to co-assort more often, we expect to observe a lower reassortment rate between them, which would manifest as small-scale similarities between phylogenetic trees of those segments.
In our case we expect SPR distances, which are proportional to the number of reassortment events that have taken place between trees, to reflect the overall (\textit{i.e} both within and between lineages) reassortment rate.

\begin{figure}[h]
	\centering		
	\includegraphics[width=0.65\textwidth]{figures/InfB_normalizedApproxSPR.png}
	\caption{\textbf{Normalized approximate SPR distances between pairs of segments.}
Following the normalization procedure approximate SPR distances are similar across all pairwise comparisons.
We interpret this as lack of evidence for small-scale topological similarities between trees of all segments, which we expect to arise if any two segments were being co-packaged and co-reassorted.
All vertical lines indicating uncertainty are 95\% highest posterior densities (HPDs).}
	\label{SPRdistances}
\end{figure}

The 95\% highest posterior density (HPD) intervals of normalized approximate SPR distances between pairs of segments encompass most means and occupy a relatively small range, suggesting there is no evidence of differences in the number of reassortments between segments (Figure \ref{SPRdistances}).
Reassortment rate given as number of SPR moves per total time in both trees shows similar results (Figure \ref{NormSPR_RErate}).
This is in line with recent experiments in influenza A that have shown that reassortment between segments differing by a single synonymous difference is highly efficient \citep{marshall2013}.
We note, however, that because of phylogenetic uncertainty our estimate of SPR distance might simply lack power.
Comparisons between independent analyses of the same segments yield distances that are comparable to distances between different segments (Figures \ref{SPRdistancesTrees} and \ref{SPRdistancesReplicates}), suggesting that phylogenetic uncertainty is making a considerable contribution to our estimates of approximate SPR distances.
Still, we find that independent replicates from the same segment (Figure \ref{SPRdistancesReplicates}) show lower SPR distances that comparisons between segments (Figure \ref{SPRdistancesTrees}), suggesting that phylogenetic noise is not completely overwhelming reassortment signal.
In addition, SPR distances themselves can only approximate (and underestimate) the actual numbers of reassortments.
Thus we caution against over-interpreting Figure \ref{SPRdistances}.
Although there might be concern about using approximate, rather than exact, SPR distances we do estimate exact SPR distances for a limited number of segment pairs - PB1, PB2 and HA - and find that after normalization exact and approximate SPR distances are not significantly different (Figures \ref{NormSPR_PB1-PB2_difference}--\ref{NormSPR_PB2-HA_difference}).


\begin{figure}
\centering  
\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_normRErate.png}
\caption{\textbf{Normalized reassortment rate}
Reassortment rate is calculated as approximate number of SPR moves per sum of total time in both trees.}
\label{NormSPR_RErate}
\end{figure}

\begin{figure}
\centering  
\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_aSPRdistances_trees.png}
\caption{\textbf{Approximate SPR distances between all pairs of trees of segments.}
There is a visible trend where comparisons between shorter segments have larger SPR distances, consistent with decreasing tree topology stability over the course of MCMC for shorter segments.}
\label{SPRdistancesTrees}
\end{figure}

\begin{figure}
\centering  
\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_aSPRdistances_replicates.png}
\caption{\textbf{Approximate SPR distances between replicate trees of each segment.}
Approximate SPR distances between replicates of MP and NS trees are much higher ($\approx$260) than any other segments, suggesting greater variability in tree topology over the course of MCMC.
SPR distances between replicates of most other segments are $\approx$200.
}
\label{SPRdistancesReplicates}
\end{figure}

\begin{figure}
\centering  
\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_NormPB1-PB2_hist.png}
\caption{\textbf{Distribution of differences between exact and approximate PB1-PB2 SPR distances after normalization.}
95\% HPD interval (blue) overlaps zero, suggesting no evidence of differences between approximate and exact SPR distances following normalization.}
\label{NormSPR_PB1-PB2_difference}
\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_NormPB1-PB2_hist2.png}
%\caption{\textbf{Distribution of ratios between exact and approximate PB1-PB2 SPR distances after normalization.}
%95\% HPD interval (blue) overlaps 1, suggesting no evidence of differences between approximate and exact SPR distances following normalization.}
%\label{NormSPR_PB1-PB2_ratio}
%\end{figure}

\begin{figure}
\centering  
\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_NormPB1-HA_hist.png}
\caption{\textbf{Distribution of differences between exact and approximate PB1-HA SPR distances after normalization.}
95\% HPD interval (blue) overlaps zero, suggesting no evidence of differences between approximate and exact SPR distances following normalization.}
\label{NormSPR_PB1-HA_difference}
\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_NormPB1-HA_hist2.png}
%\caption{\textbf{Distribution of ratios between exact and approximate PB1-HA SPR distances after normalization.}
%95\% HPD interval (blue) overlaps 1, suggesting no evidence of differences between approximate and exact SPR distances following normalization.}
%\label{NormSPR_PB1-HA_ratio}
%\end{figure}

\begin{figure}
\centering  
\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_NormPB2-HA_hist.png}
\caption{\textbf{Distribution of differences between exact and approximate PB2-HA SPR distances after normalization.}
95\% HPD interval (blue) overlaps zero, suggesting no evidence of differences between approximate and exact SPR distances following normalization.
Due to excessively long computation time of exact SPR distances between PB2 and HA trees few comparisons were made.}
\label{NormSPR_PB2-HA_difference}
\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_NormPB2-HA_hist2.png}
%\caption{\textbf{Distribution of ratios between exact and approximate PB2-HA SPR distances after normalization.}
%95\% HPD interval (blue) overlaps 1, suggesting no evidence of differences between approximate and exact SPR distances following normalization.
%Due to excessively long computation time of exact SPR distances between PB2 and HA trees few comparisons were made.}
%\label{NormSPR_PB2-HA_ratio}
%\end{figure}

\begin{figure}
\centering  
\includegraphics[width=0.95\textwidth]  {supp_figures/InfB_LDheatmap.1600.aa.ChiSqdf.minorCutoff1percent.png}
\caption{\textbf{Heatmap of genome-wide linkage disequilibrium ($\chiSq$) between polymorphic amino acid sites.}
Patterns of LD across the genome suggest a network of reciprocally linked amino acid sites on PB1, PB2, HA and, to some extent NA, proteins.
Proximity of sites on heatmaps might not correspond to proximity of sites within proteins.}
\label{ChiGenome}
\end{figure}

\begin{figure}
\centering  
\includegraphics[width=0.85\textwidth]  {supp_figures/InfB_LDheatmap.PB1_PB2_HA.1600.aa.ChiSqdf.minorCutoff1percent.png}
\caption{\textbf{Heatmap of linkage disequilibrium ($\chiSq$) between amino acid sites on PB1, PB2 and HA proteins.}
Numbers next to each row and column correspond to amino acid site number within a given protein starting from methionine.
Amino acid sites exhibiting reciprocally high LD between PB1, PB2 and HA proteins are: 176, 211, 213, 508 (PB1), 115, 399, 470, 642 (PB2) and 96, 163, 520 and 570 (HA).
Sites 211 and 213 on the PB1 protein are very close to each other and the stretch of sequence around these residues contains many positively charged amino acids (lysine and arginine).
Multiple nuclear localization signals (NLSs) are predicted to occur around this region and sites 211 and 213 are either predicted to be near the end of a mono-partite NLS or the beginning of a bi-partite NLS.
Previous research \citep{nath1990} suggests that in the influenza A PB1 protein residue 211 (homologous to influenza B PB1 residue 211) is the last residue of a bi-partite NLS.
Almost all Yamagata lineage isolates possess arginine (R) residue at PB1 position 211 and a serine (S) residue at position 213, whereas Victoria lineage isolates have lysine (K) at position 211 and threonine (T) at position 213.
It remains to be seen whether these sites significantly affect the nuclear import efficiency of the PB1 protein of either lineage.
Though the PB1 protein is known to accumulate in the nucleus on its own, efficient import into the nucleus requires the presence of the PA protein \citep{fodor2004}.
Similarly, site 399 on the PB2 protein are close to residues 377, 406 and 408 which are homologous to sites in influenza A that are responsible for mRNA cap-binding \citep{guilligay2008}.}
\label{ChiCore}
\end{figure}

\begin{figure}
\centering  
\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_robustCounting.png}
\caption{\textbf{Synonymous, non-synonymous and nucleotide substitution rates in segments under different PB1-PB2-HA complexes.}
Evolutionary rate dissimilarities under Vic and Yam PB1-PB2-HA complexes are not systematic and appear negligible.
Synonymous and non-synonymous rates were calculated by dividing the sum of all substitutions of a given class by the total amount of evolutionary time under each PB1-PB2-HA constellation.
Nucleotide rates were calculated by multiplying the inferred nucleotide substitution rate on each branch by the branch length, then dividing this by the total amount of evolutionary time under each PB1-PB2-HA constellation.
Vertical bars indicating uncertainty are 95\% HPDs, black bars indicate 95\% HPDs that do not overlap.}
\label{robustCounting}
\end{figure}

%Figures \ref{SPRdistancesTrees} and \ref{SPRdistancesReplicates} are showing approximate SPR distances prior to the normalization procedure (see Methods).
%Figure \ref{SPRdistancesTrees} shows that MP and NS trees show greater approximate SPR distances when compared to any other segment tree, which is consistent with the fact that only M1 and NS1 coding sequences, which are shorter than any %other coding sequences, were used to infer the trees.
%Figure \ref{SPRdistancesReplicates} confirms this by showing greater variability in tree topology (as greater approximate SPR distance) when MP and NS trees are compared against their replicates.
%For all other segments approximate SPR distances between all pairs of trees prior to normalization suggests that most segment pairs have similar posterior distributions of approximate SPR distances.

%Figures \ref{deltaTMRCAtrees} and \ref{deltaTMRCAreplicates} show $\dtmrca$ statistics for inter-segment and intra-segment, respectively, comparisons prior to normalization.
%Figure \ref{deltaTMRCAtrees} shows that similarity in TMRCAs (as low $\dtmrca$) between PB1, PB2 and HA segments is due to considerably smaller $\dtmrca$ statistic for the comparison between these segments.
%It is also noteworthy that NA and MP comparisons have similarly low values for this statistic.
%Figure \ref{deltaTMRCAreplicates} shows that TMRCAs are quite robust over the course of MCMC and node times do not, even in the most extreme cases, deviate from their replicates by more than 1 year.

\begin{figure}
\centering  
\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_deltaTMRCA_trees.png}
\caption{\textbf{Mean $\dtmrca$ between all pairs of trees of segments.}
Mean $\dtmrca$ between trees of segments reveal that tip pairs in PB1, PB2 and HA trees have very similar TMRCAs.
The upper tail of the 95\% HPD (HPDs are represented as vertical lines) interval of mean $\dtmrca$ values for PB1-PB2-HA and MP-NA trees do not exceed 3 years.}
\label{deltaTMRCAtrees}
\end{figure}

\begin{figure}
\centering  
\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_deltaTMRCA_replicates.png}
\caption{\textbf{Mean $\dtmrca$ between replicate trees of each segment.}
Mean $\dtmrca$ values between independent analyses of each segment show that mean $\dtmrca$ values rarely exceed 1 year.}
\label{deltaTMRCAreplicates}
\end{figure}

%We also tested the accuracy of approximate SPR distances using exact SPR distances calculated from PB1, PB2 and HA comparisons.
%Figures \ref{SPR_PB1-PB2_difference}--\ref{SPR_PB2-HA_ratio} show that after normalization the approximate and exact SPR distance estimates are not significantly different.
%There is also good correlation between the two SPR distances (Figures \ref{SPR_PB1-PB2_correlation}--\ref{SPR_PB2-HA_correlation}). 
%Due to excessively long computation time of exact SPR distances very few PB2--HA comparisons were made.

%\begin{figure}
%\centering  
%\includegraphics[width=0.5\textwidth]  {supp_figures/InfB_supp_NormPB1-PB2_corr.png}
%\caption{\textbf{Correlation between exact and approximate PB1-PB2 SPR distances after normalization.}}
%\label{NormSPR_PB1-PB2_correlation}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.5\textwidth]  {supp_figures/InfB_supp_NormPB1-HA_corr.png}
%\caption{\textbf{Correlation between exact and approximate PB1-HA SPR distances after normalization.}}
%\label{NormSPR_PB1-HA_correlation}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.5\textwidth]  {supp_figures/InfB_supp_NormPB2-HA_corr.png}
%\caption{\textbf{Correlation between exact and approximate PB2-HA SPR distances after normalization.}
%Due to excessively long computation time of exact SPR distances between PB2 and HA trees few comparisons were made.}
%\label{NormSPR_PB2-HA_correlation}
%\end{figure}

%Figures \ref{SPR_PB1-PB2_difference}--\ref{SPR_PB2-HA_ratio} focus on differences between exact and approximate SPR distances and show that prior to normalization approximate SPR distances are overestimates of exact SPR distances.
%Figures \ref{SPR_PB1-PB2_correlation}--\ref{SPR_PB2-HA_correlation} show that there is a similar degree of correspondence between approximate and exact SPR distances to when approximate and exact SPR distances are normalized (Figures %\ref{NormSPR_PB1-PB2_correlation}--\ref{NormSPR_PB2-HA_correlation}) using the procedure described in Methods.

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_PB1-PB2_hist.png}
%\caption{\textbf{Distribution of differences between exact and approximate PB1-PB2 SPR distances.}}
%\label{SPR_PB1-PB2_difference}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_PB1-PB2_hist2.png}
%\caption{\textbf{Distribution of ratios between exact and approximate PB1-PB2 SPR distances.}}
%\label{SPR_PB1-PB2_ratio}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_PB1-HA_hist.png}
%\caption{\textbf{Distribution of differences between exact and approximate PB1-HA SPR distances.}}
%\label{SPR_PB1-HA_difference}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_PB1-HA_hist2.png}
%\caption{\textbf{Distribution of ratios between exact and approximate PB1-HA SPR distances.}}
%\label{SPR_PB1-HA_ratio}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_PB2-HA_hist.png}
%\caption{\textbf{Distribution of differences between exact and approximate PB2-HA SPR distances.}}
%\label{SPR_PB2-HA_difference}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_supp_PB2-HA_hist2.png}
%\caption{\textbf{Distribution of ratios between exact and approximate PB2-HA SPR distances.}}
%\label{SPR_PB2-HA_ratio}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.4\textwidth]  {supp_figures/InfB_supp_PB1-PB2_corr.png}
%\caption{\textbf{Correlation between exact and approximate PB1-PB2 SPR distances.}}
%\label{SPR_PB1-PB2_correlation}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.4\textwidth]  {supp_figures/InfB_supp_PB1-HA_corr.png}
%\caption{\textbf{Correlation between exact and approximate PB1-HA SPR distances.}}
%\label{SPR_PB1-HA_correlation}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.4\textwidth]  {supp_figures/InfB_supp_PB2-HA_corr.png}
%\caption{\textbf{Correlation between exact and approximate PB2-HA SPR distances.}}
%\label{SPR_PB2-HA_correlation}
%\end{figure}

%\clearpage
%\section*{Multidimensional scaling}
%Figures \ref{MDSaaLD} and \ref{MDSntLD} visualize the relationships between segments based on mean amino acid and nucleotide site linkage disequilibrium estimates, respectively.
%PB1, PB2 and HA segments appear to be by far the most similar segments, as indicated by their proximity in the MDS embedding.
%Figures \ref{MDSaaCorr} and \ref{MDSntCorr} compare the observed mean LD values between segments. 
%Based on mean amino acid site LD (Figure \ref{MDSaaCorr}) only PB1, PB2 and HA segments show considerable clustering, all other segments being equidistant from each other in the MDS embedding.
%MDS distance versus mean nucleotide site LD scatter plot, on the other hand, shows a continuum of relationships between segments, which is probably caused by more variability at the nucleotide level than the amino acid level.
%This difference in variability would build up linkage disequilibrium between nucleotide sites and would reduce it via convergent substitutions over relatively short periods of time.

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_8x8_aaLD_MDS.png}
%\caption{\textbf{2-dimensional MDS embedding of influenza B virus segments based on mean amino acid linkage disequilibrium ($\chiSq$).}}
%\label{MDSaaLD}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_aaLD_MatrixMDScorr.png}
%\caption{\textbf{Correlation between mean amino acid site linkage disequilibrium and MDS embedding distance for all pairs of segments.}}
%\label{MDSaaCorr}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_8x8_ntLD_MDS.png}
%\caption{\textbf{2-dimensional MDS embedding of influenza B virus segments based on mean nucleotide linkage disequilibrium ($\chiSq$).}}
%\label{MDSntLD}
%\end{figure}

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_ntLD_MatrixMDScorr.png}
%\caption{\textbf{Correlation between mean nucleotide site linkage disequilibrium and MDS embedding distance for all pairs of segments.}}
%\label{MDSntCorr}
%\end{figure}

%\section*{Differences between Vic and Yam PB1-PB2-HA}
%Through the combination of inferred PB1-PB2-HA background and realized synonymous and non-synonymous substitution counts on branches \citep{obrien2009} it was possible to investigate whether Victoria and Yamagata lineage backgrounds differ in substitution rates.
%Figure \ref{robustCounting} shows the inferred synonymous, non-synonymous (both as substitutions per year) and overall nucleotide substitution rate (as substitutions per site per year) under different lineages of PB1-PB2-HA.

%\begin{figure}
%\centering  
%\includegraphics[width=0.65\textwidth]  {supp_figures/InfB_lineageRatios.png}
%\caption{\textbf{Total counts and ratios of Victoria and Yamagata lineage sequences in each segment.}}
%\label{lineageRatios}
%\end{figure}

\bibliographystyle{mbe}
\bibliography{fluB}

\end{document}
